src/lxc/tools/lxc_top.c

```diff
@@ -204,7 +212,6 @@ static uint64_t stat_get_int(struct lxc_container *c, const char *item)
 
 	len = c->get_cgroup_item(c, item, buf, sizeof(buf));
 	if (len <= 0) {
-		fprintf(stderr, "Unable to read cgroup item %s\n", item);
```

Comment:
can we keep these error messages? I think it might be very-very useful in case if something goes wrong.